src/Widgets/ImageWidget/ImageWidgetEditingConfig.js

```diff
@@ -24,9 +24,22 @@ Scrivito.provideEditingConfig("ImageWidget", {
       title: "Link (optional)",
       description: "The link where this image should lead.",
     },
+    animation: {
+      title: "Animation (optional)",
+      description: "Animation effect that will be added to the photo",
```

Comment:
How about this? `What animation should be shown, when this image becomes visible?`